common/persistence/pinot/pinot_visibility_store.go

```diff
@@ -763,7 +763,7 @@ func (v *pinotVisibilityStore) getCountWorkflowExecutionsQuery(tableName string,
 	comparExpr, err := v.pinotQueryValidator.ValidateQuery(comparExpr)
 	if err != nil {
 		v.logger.Error(fmt.Sprintf("pinot query validator error: %s", err))
-	}
+	} // can't test because it's hard to create a scenario where this error happens
```

Comment:
we could mock some things inside `ValidateQuery` to mimic this failure but I don't think it's crucial